### PR TITLE
ENH: add direct_img_list producing a 10x10 list, not np.ndarray

### PR DESCRIPTION
**Pull request by mrakitin**
This depends on https://github.com/NSLS-II/ophyd/pull/682/commits/a38fa6f3b1231fadd67b0ddd2d417e5cd3bfde59.

### CODE REVIEW

File: suitcase/utils/tests/conftest.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def _internal_plan(dets):
     yield from _internal_plan(dets)
 
 
-@pytest.fixture(params=['det', 'direct_img', 'det direct_img'],
+@pytest.fixture(params=['det', 'direct_img', 'direct_img_list',
+                        'det direct_img'],
```
Comment (awalter-bnl) [inline suitcase/utils/tests/conftest.py]:
```suggestion
                        'det direct_img direct_img_list'],
```

Comment (awalter-bnl) [reply suitcase/utils/tests/conftest.py]: perhaps we should also add the new detector to the 'group' pass for the detectors

**Comment by awalter-bnl** [comment]
FYI the failure here is because we need to change the 'name' field in the ophyd PR#682 line 1262. IT occurs because when testing with both direct_img and direct_img_list as detectors simultaneously there is currently a name clash.

**Comment by mrakitin** [comment]
Thanks @awalter-bnl, I hope https://github.com/NSLS-II/ophyd/pull/682/commits/bebb6177d6bb87ed990fa4ce8ff429244e332153 will resolve the issue.

**Comment by mrakitin** [comment]
It's now passing with the updates `name`s of the detectors.

**Pull request merged**